### ISSUE TITLE
Error: Plugin nexus-prisma failed on "onInstall" hook:

### ISSUE THREAD

**Issue by ZaneH** [graphql-nexus/nexus-plugin-prisma#502]
Hey Prisma team, love the product.

I've been working on a project since the 2.0 Prisma Preview became available and things have been going well. I ran into an issue last Friday, that has bled into Monday and I'm curious if anyone has any suggestions to get back to a working project.

I believe I updated from b13.2 to b14.2, added the new dependencies, setup nexus-prisma in `server.ts` as per the example projects. And I also made the necessary changes to my `schema.prisma` file and `package.json`.

Here's the error after running `yarn dev`:
``` bash
$ yarn dev
yarn run v1.19.0
$ ts-node-dev --no-notify --respawn --transpileOnly src/server
Using ts-node version 8.4.1, typescript version 3.6.4
Error: Plugin nexus-prisma failed on "onInstall" hook:

Error: Could not find photon package at @generated/photon. Check that you have configured your Photon generator block in schema.prisma correctly and run prisma generate.
    at Object.exports.get (/Users/zane/mono-repo/node_modules/nexus-prisma/src/dmmf/utils.ts:14:11)
    at new SchemaBuilder (/Users/zane/mono-repo/node_modules/nexus-prisma/src/builder.ts:162:38)
    at Object.build (/Users/zane/mono-repo/node_modules/nexus-prisma/src/builder.ts:89:19)
    at Object.onInstall (/Users/zane/mono-repo/node_modules/nexus-prisma/src/index.ts:43:16)
    at Object.triggerOnInstall (/Users/zane/mono-repo/node_modules/nexus/src/plugins.ts:113:36)
    at /Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1481:22
    at Array.forEach (<anonymous>)
    at buildTypesInternal (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1480:21)
    at makeSchemaInternal (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1528:9)
    at Object.makeSchema (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1579:9)
    at Object.triggerOnInstall (/Users/zane/mono-repo/node_modules/nexus/src/plugins.ts:115:15)
    at /Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1481:22
    at Array.forEach (<anonymous>)
    at buildTypesInternal (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1480:21)
    at makeSchemaInternal (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1528:9)
    at Object.makeSchema (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1579:9)
    at Object.<anonymous> (/Users/zane/mono-repo/packages/prisma/src/schema.ts:6:23)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Module._compile (/Users/zane/mono-repo/node_modules/source-map-support/source-map-support.js:521:25)
    at Module.m._compile (/private/var/folders/x2/06njp_z52rxg1d5s5chc6m2m0000gn/T/ts-node-dev-hook-013590212857752304.js:57:25)
[ERROR] 16:27:33 Error: Plugin nexus-prisma failed on "onInstall" hook:

Error: Could not find photon package at @generated/photon. Check that you have configured your Photon generator block in schema.prisma correctly and run prisma generate.
    at Object.exports.get (/Users/zane/mono-repo/node_modules/nexus-prisma/src/dmmf/utils.ts:14:11)
    at new SchemaBuilder (/Users/zane/mono-repo/node_modules/nexus-prisma/src/builder.ts:162:38)
    at Object.build (/Users/zane/mono-repo/node_modules/nexus-prisma/src/builder.ts:89:19)
    at Object.onInstall (/Users/zane/mono-repo/node_modules/nexus-prisma/src/index.ts:43:16)
    at Object.triggerOnInstall (/Users/zane/mono-repo/node_modules/nexus/src/plugins.ts:113:36)
    at /Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1481:22
    at Array.forEach (<anonymous>)
    at buildTypesInternal (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1480:21)
    at makeSchemaInternal (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1528:9)
    at Object.makeSchema (/Users/zane/mono-repo/node_modules/nexus/src/builder.ts:1579:9)
```

**Comment by IMax153** [comment]
Hey @ZaneH were you able to fix the issue? I am running into the same error and wondering how I should go about trying to troubleshoot.

**Comment by ZaneH** [comment]
@IMax153 Yes, I’ve gotten past it, but I am facing a new issue: https://github.com/prisma/prisma2/issues/826

I’m not 100% how I solved this, but I have a few ideas:
* Make sure your project’s prisma version matches the CLI version. `prisma2 --version`
* Follow the example projects to make sure your foundation is 100% correct. I had to change some of my architecture to match the examples
* Try the postinstall script and the command line version of running `prisma2 generate` (they behave differently for me)
* Lastly, make sure you have the compatible versions of nexus-prisma, nexus, etc. within your package.json. I used the example projects for reference.

Example projects: https://github.com/prisma/prisma-examples/tree/prisma2/typescript

(These seems to be updated fairly quickly ^)
